### PR TITLE
Add the ability to auth via a JWT

### LINKED ISSUE #7: Demonstrate file uploads using CURL

### PR DESCRIPTION
**Pull request by jkeck**
Closes #7

Added an example `curl` command in the upload pages that uses the generated JWT

Screen Shot 2020-10-29 at 2 43 30 PM

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+jwt:
+  secret: 'i♥️dolphins'
```
Comment (jkeck) [inline config/settings.yml:2]: 🐬

File: app/models/ability.rb [hunk 2]
```diff
@@ -4,8 +4,26 @@
...
+                          Settings.jwt.secret,
+                          Settings.jwt.algorithm
+                        )[0]
+                      else
+                        {}
+                      end
+
+      can :read, Organization, allowlisted_jwts: { jti: token_payload['jti'] }
+
+      can %i[create read update], [Stream, Upload, Batch], organization: { allowlisted_jwts: { jti: token_payload['jti'] } }
```
Comment (jkeck) [inline app/models/ability.rb:23]: Not sure what other permissions we might want in this area, but I imagine we can iterate.

**Review by mejackreed — APPROVED** [review]
Thanks I was able to get this going locally. Nice job on it

**Pull request merged**